src/solvers/dg_simplices/fluxdiff.jl

```diff
@@ -0,0 +1,182 @@
+"""
+  function hadamard_sum_ATr!(du, ATr, volume_flux, u, skip_index=(i,j)->false)
+
+Computes the flux difference ∑_j Aij * f(u_i, u_j) and accumulates the result into `du`. 
+- `du`, `u` are vectors
+- `ATr` is the transpose of the flux differencing matrix `A`. The transpose is used for 
+faster traversal since matrices are column major in Julia. 
+"""
+@inline @muladd function hadamard_sum_ATr!(du, ATr, volume_flux::VF, u, skip_index=(i,j)->false) where {VF}
+  rows,cols = axes(ATr)
+  for i in cols
+    ui = u[i]
+    val_i = du[i]
+    for j in rows
+      if !skip_index(i,j)                
+          val_i += ATr[j,i] * volume_flux(ui, u[j])
+      end
+    end
+    du[i] = val_i 
+  end
+end
+
+function build_lazy_physical_derivative(elem::Int, orientation::Int, mesh::VertexMappedMesh{2, Tri}, dg, cache) 
```

Comment:
Could you please avoid this abbreviation and use the full name `element`?
```suggestion
function build_lazy_physical_derivative(element::Int, orientation::Int, mesh::VertexMappedMesh{2, Tri}, dg, cache) 
```

Reply:
Can do - will look for these and change them.